runtime/near-vm-runner/src/runner.rs

```diff
@@ -24,6 +23,26 @@ use std::sync::Arc;
 /// validators, even when a guest error occurs, or else their state will diverge.
 pub(crate) type VMResult<T = VMOutcome> = Result<T, VMRunnerError>;
 
+#[tracing::instrument(target = "vm", level = "debug", "prepare", skip_all, fields(
+    code.hash = %ext.code_hash(),
+    method_name,
+    vm_kind = ?wasm_config.vm_kind,
+    burnt_gas = tracing::field::Empty,
+    compute_usage = tracing::field::Empty,
+))]
+pub fn prepare(
+    ext: &(dyn External + Send),
```

Comment:
This actually no longer requires `Send` from this API specifically, but `External` will need to be sendable if we want to allow preparing in a different thread anyway. So I'm keeping this bound here to make sure implementors of this type don't go rogue...